packages/dev/core/src/Cameras/arcRotateCamera.ts

```diff
@@ -1207,8 +1207,11 @@ export class ArcRotateCamera extends TargetCamera {
         meshes = meshes || this.getScene().meshes;
 
         const minMaxVector = Mesh.MinMax(meshes);
-        const distance = Vector3.Distance(minMaxVector.min, minMaxVector.max);
+        let distance = this._calculateLowerRadiusFromModelBoundingSphere(minMaxVector.min, minMaxVector.max);
 
+        // If there are defined limits, we need to take them into account
+        distance = Math.max(distance, this.lowerRadiusLimit ?? 0);
+        distance = Math.min(distance, this.upperRadiusLimit ?? Number.MAX_VALUE);

```

Comment:
nit - can probably be a single call instead of changing the variable twice. Not a must though :-)

Also - in modern JS `??` makes a LOT of sense, but in ES5 (which we still use in UMD), `??` is expanded to a lot of code. It is better to use `||` is possible.

Reply:
Easy enough to do

Reply:
Change has been made